driver/bpf/fillers.h

```diff
@@ -190,6 +190,12 @@ FILLER_RAW(terminate_filler)
 					++state->n_drops_buffer_other_interest_enter;
 				}
 				break;
+			case PPME_PROCEXIT_E:
```

Comment:
@Andreagit97 while we are here, I think it was during the loginuid PR where I saw we could cleanup and remove some of the old `PPME_SYSCALL_CLONE_*` events. Could we sneak this cleanup in?

Also do you think the categories `n_drops_buffer_dir_file*` and `n_drops_buffer_other_interest*` could benefit from some updates while here?

Reply:
Done :) 

> Also do you think the categories n_drops_buffer_dir_file* and n_drops_buffer_other_interest* could benefit from some updates while here?

During my experiments I've never used other stats so at the moment I would say no :thinking: